static/js/impala/global.js

```diff
@@ -167,6 +171,8 @@ function initBanners(delegate) {
             z.visitor.set('seen_acr_pitch', 1);
         } else if ($parent.is('#appruntime-pitch')) {
             z.visitor.set('seen_appruntime_pitch', 1);
+        } else if ($parent.is('#fxa-migration-banner')) {
+            z.visitor.set('seen_fxa_migration_banner', 1);
```

Comment:
They won't see it until `localStorage` gets cleared. I'll put it in `sessionStorage` instead.